po/fr.po

```diff
@@ -14589,6 +14587,40 @@ msgid ""
 "\n"
 "Click on \"Next\" to proceed or \"Cancel\" to Abort Import."
 msgstr ""
+"Cet assistant vous aidera à importer des Prix à partir d'un fichier CSV.\n"
+"\n"
+"Un nombre minimal de colonnes doivent être présentes pour que l'importation "
+"réussisse : il s'agit de Date, Montant, Espace de nom d'origine, Symbole "
+"d'origine et Devise de destination. Si toutes les entrées sont pour le même "
+"Produit /  Devise, vous pouvez les sélectionner et les colonnes seront Date "
+"et Montant.\n"
+"\n"
+"Plusieurs options existent pour spécifier le délimiteur ainsi qu'une option "
+"de largeur fixe. Avec l'option largeur fixe, double-cliquez sur les lignes "
+"affichées pour définir la largeur de la colonne, puis faites un clic-droit "
+"pour la modifier si nécessaire.\n"
+"\n"
+"Exemples : \"FTSE\",\"RR.L\",\"21/11/2016\",5.345,\"GBP\" and "
+"CURRENCY;USD;2016-11-21;1.56;GBP\n"
```

Comment:
@thjungers How about replacing Rolls-Royce, London by PSA, Paris and GBP by EUR?